pkg/shell/machines/machines.js

```diff
@@ -42,12 +42,31 @@ export function host_superuser_storage_key(host) {
         return null;
 }
 
-export function get_host_superuser_value(host) {
-    const key = host_superuser_storage_key(host);
+export function get_init_superuser_for_options(options) {
+    let value = null;
+    const key = host_superuser_storage_key(options.host);
     if (key)
-        return window.localStorage.getItem(key);
-    else
-        return null;
+        value = window.localStorage.getItem(key);
+
+    /* When connecting, we can optionally try to start a privileged
+     * bridge immediately.  However, it is quite likely that that
+     * needs a password and if we don't have one, it will likely fail.
+     * That would be okay, but sudo is very noisy about failures and
+     * might send nasty emails to your parents.  For that reason we
+     * always pass "init-superuser": "none" here.
```

Comment:
This needs improvement: " For that reason we pass "init-superuser": "none" here when there is no password."

Reply:
Upps, forgot about this.  Fixed now.